### PR TITLE
Video component

### PR DESCRIPTION
**Pull request by fredjens**

### CODE REVIEW

File: scrollbar/.gitignore
Comment (apers) [inline scrollbar/.gitignore:1]: Can remove this as there is one in the root of the repo

File: scrollbar/.prettierrc.json
Comment (apers) [inline scrollbar/.prettierrc.json:1]: Can remove this as there is one in the root of the repo

File: scrollbar/README.md
Comment (apers) [inline scrollbar/README.md:1]: Delete this as well

File: video/.gitignore
Comment (apers) [inline video/.gitignore:1]: Can remove this as there is one in the root of the repo

File: video/.prettierrc.json
Comment (apers) [inline video/.prettierrc.json:1]: Can remove this as there is one in the root of the repo

File: video/README.md
Comment (apers) [inline video/README.md:1]: Delete

**Review by apers — APPROVED** [review]

File: scrollbar/.prettierrc.json
Comment (fredjens) [reply scrollbar/.prettierrc.json:1]: Hm, are they coming from `vev create` maybe 🤔

**Pull request merged**